src/vendor/wordpress/_wordpress.scss

```diff
@@ -0,0 +1,24 @@
+@use "../../mixins/button";
+@use "../../design-tokens/sizes.yml";
+
+/**
+ * Gutenberg block: Button
+ * Applies our pattern library styles to buttons generated via Gutenberg blocks.
+ *
+ * 1. The block editor allows editors to adjust button border radius. This style
+ *    is applied via an inline style on the `wp-block-button__link` element. In
+ *    order to prevent unwanted border-radius we apply an `!important` tag.
+ */
+
+.wp-block-button {
+  display: inline-block;
+
+  .wp-block-button__link {
+    @include button.default;
+    border-radius: sizes.$radius-medium !important; /* 1 */
```

Comment:
If this style is omitted, what happens? Is it always overridden or is it only overridden if the author chooses a different radius?

Reply:
Great question @tylersticka, 

If the author does not customize the border-radius section in the block editor this style is not needed. 

If the author does does modify the border-radius settings, this style is needed to prevent pill shaped or other unwanted border radius values. 

Does that answer your question?

Reply:
It does.

In that case, I'm not sure I agree that this style is necessary. It feels odd to me that we would purposely break CMS functionality from our stylesheet instead of disabling the CMS feature somehow.

Reply:
Excellent point. 

I had considered this as well. However, my current understanding is disabling the border-radius option is either not possible, or rather difficult. 

https://github.com/WordPress/gutenberg/issues/19796

Reply:
I am comfortable with the worse-case scenario of "the Cloud Four team may optionally select an off-brand border radius for buttons."